### ISSUE TITLE
TypeError: Expected object_relevance_per_qid dictionary and measures set.

### ISSUE THREAD

**Issue by BtlWolf** [embeddings-benchmark/mteb#1119]
**I used the test cases in the official tutorial：**

![image](https://github.com/user-attachments/assets/2450445e-d1bc-4348-b448-82cfa8d887e2)

**but encountered an error on the ArguAna dataset：**

![image](https://github.com/user-attachments/assets/6f62b7fb-92b2-4e28-81af-479b3e6a7175)

**What could be the cause of this?**

**Comment by KennethEnevoldsen** [comment]
What version are you on?

I can't reproduce the error using:
```
tasks = mteb.get_tasks(tasks=["ArguAna"])
model = mteb.get_model("sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2")
evaluation = mteb.MTEB(tasks=tasks, task_langs=["en"]) 
evaluation.run(model, overwrite_results=True)
```

**Comment by BtlWolf** [comment]
> What version are you on? It might be that a simple update to the latest version will fix it.
> 
> I can't reproduce the error using:
> 
> ```
> tasks = mteb.get_tasks(tasks=["ArguAna"])
> model = mteb.get_model("sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2")
> evaluation = mteb.MTEB(tasks=tasks, task_langs=["en"]) 
> evaluation.run(model, overwrite_results=True)
> ```

Thank you，I resolved this issue by downgrading to version 1.12.79.

**Comment by KennethEnevoldsen** [comment]
I tried to reproduce it on the latest version of the main as well. I can't seem to reproduce it there, either.

Feel free to re-open this issue if you experience it again, but will close it for now
